lib/public/views/Environments/Details/EnvironmentDetailsPage.js

```diff
@@ -75,10 +78,9 @@ export const EnvironmentDetailsPage = ({ envs: { detailsModel } }) => detailsMod
                 ]),
                 h('#environment-creation-date', [h('strong', 'Created at '), formatTimestamp(createdAt)]),
             ]),
-            (flpInfologgerLinkUrl || epnInfologgerLinkUrl) && h('h3.flex-row.w-100.g2.items-baseline.mb3', [
-                h('', 'Infologger: '),
-                flpInfologgerLinkUrl && h('a', { href: flpInfologgerLinkUrl }, 'FLP'),
-                epnInfologgerLinkUrl && h('a', { href: epnInfologgerLinkUrl }, 'EPN'),
+            h('h3.flex-row.w-100.g2.items-baseline.mb3', [
+                (flpInfologgerLink || epnInfologgerLink) && [h('', 'Infologger: '), flpInfologgerLink, epnInfologgerLink],
+                status === 'RUNNING' && aliEcsEnvironmentLink && [h('', '-'), aliEcsEnvironmentLink],
             ]),
```

Comment:
Here you probably need a flatMap (like https://github.com/AliceO2Group/Bookkeeping/blob/f1432d6fb2bbfcc5604d50f0e4da30a50e65c872/lib/public/components/environments/environmentStatusHistoryComponent.js#L39) because if there is no infologger links, you will have an issue

Reply:
applied

Reply:
No. Please carefully look at how it is implemented in the example I sent.

Reply:
It because I don't think exactly the same approach is better, but ok applied